Makefile

```diff
@@ -14,10 +14,17 @@ ifeq (${CI}, true)
 	endif
 endif
 
+PODMAN_DEPENDENCY = podman
+ifeq (${CI}, true)
+	ifneq (${TEST_CGROUPSV2}, true)
```

Comment:
I think you can move this into above `ifeq (${CI}, true)`

https://github.com/kata-containers/tests/blob/3fdb4c3a8259f91fd466868a7e39f31a4c16a856/Makefile#L10-L15

Reply:
done